src/ui/setup/prefs.ts

```diff
@@ -155,7 +155,7 @@ export async function isLocalNagDismissed(nag: LocalNag) {
     return;
   }
 
-  const replaySessions = await getReplaySessions();
+  const replaySessions = await getReplaySession(recordingId);
   const replaySession = replaySessions[recordingId];
```

Comment:
This looks a little suspicious now:
```js
const replaySessions = await getReplaySession(recordingId);
const replaySession = replaySessions[recordingId];
```

Given the names and params, I would expect
```js
const replaySession = await getReplaySession(recordingId);
```

Reply:
Yep, you are right